rai/node/node.cpp

```diff
@@ -3559,7 +3559,7 @@ void rai::active_transactions::announce_votes ()
 				unconfirmed_announcements += i->announcements;
 			}
 			node.background ([election_l]() { election_l->broadcast_winner (); });
-			if (i->announcements % announcement_min == 2)
+			if (i->announcements % announcement_min == 1)
```

Comment:
May be increase to
`if (i->announcements % (announcement_min * 2) == 1)`
?

Reply:
what about `if (i->announcements % 4 == 0)` that way the loop is entered every 4th run

Reply:
I guess better have == 1 or == 2 so new block can propagate in network before confirmation request. Not sure